com.unity.multiplayer.mlapi/Runtime/Profiling/ITransportProfilerData.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace MLAPI.Profiling
+{
+    public interface ITransportProfilerData
+    {
+        Dictionary<string, int> GetTransportGetData();
```

Comment:
Confused by this method name. Should it be something like `GetTransportProfilerData` ?